shpc/main/registry/remote.py

```diff
@@ -190,8 +185,11 @@ def _update_cache(self, force=False):
         if self._cache and not force:
             return
 
+        library_url = self.library_url
```

Comment:
If this is a property my preference is to reference it directly.

Reply:
Yes it's a property, but the value needs to be used there and a couple of lines below. Since properties are not cached by default, I  thought I'd essentially make a local cache by keeping the value in the local variable `library_url`

Reply:
Gotcha. So let's instead add self._library_url to do that caching, so when you call the property it checks for that first and returns it if it's defined, otherwise sets it and then returns it. Properties should be able to be used like that, and without copying them as we see here (it's not expected).

Reply:
Changed and pushed (together with the other changes I did earlier today and forgot to push),